### PR TITLE
Product updates: add API branch management entry

### PR DESCRIPTION
**Pull request by saro0h**
This is a proposition of product update for the new features about [Branch Management](https://www.notion.so/bumpsh/Branch-management-thanks-to-API-8ebaa14cb3994ba19715ac04d0ed2476?pvs=4).

I need a good visual to make look nice :) (@antlio ?)

### CODE REVIEW

**Comment by paulRbr** [comment]
> I'm sorry, I don't know yet how to generate a preview :/

don't worry it's generated automatically on the PR. Cf [netlify's comment](https://github.com/bump-sh/docs/pull/74#issuecomment-1580470734) :slightly_smiling_face:.

File: docs/product-updates/2023-06-07-api-branch-management.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: Simplify Your API Workflow with Bump.sh's API New Branch Management Feature
+tags: [New feature]
```
Comment (paulRbr) [inline docs/product-updates/2023-06-07-api-branch-management.md:3]: :tipping_hand_person: suggestion: I would mention the API as a general tag for this update as it's only changing the Bump API.

```suggestion
tags: [New feature, Bump API]
```

What do you think?

File: docs/product-updates/2023-06-07-api-branch-management.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+image: /files/changelog/temporary-pic.png
+---
+
+[NEED A VISUAL HERE]
+
+We released our latest feature: Branch Management with the [Bump API](https://developers.bump.sh/).
+
+Designed to enhance your API development process, these new endpoints empower API  ecosystem. With the introduction of branch creation, listing, setting default branche, and branch deletion. Bump.sh revolutionizes how you deliver ephemeral versions and begin new branches with major versions. Let's explore how this new feature simplifies your API workflow.
```
Comment (paulRbr) [inline docs/product-updates/2023-06-07-api-branch-management.md:11]: mini-typo: 

```suggestion
Designed to enhance your API development process, these new endpoints empower API  ecosystem. With the introduction of branch creation, listing, setting default branch, and branch deletion. Bump.sh revolutionizes how you deliver ephemeral versions and begin new branches with major versions. Let's explore how this new feature simplifies your API workflow.
```

File: docs/product-updates/2023-06-07-api-branch-management.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+title: Simplify Your API Workflow with Bump.sh's API New Branch Management Feature
+tags: [New feature]
+image: /files/changelog/temporary-pic.png
+---
+
+[NEED A VISUAL HERE]
+
+We released our latest feature: Branch Management with the [Bump API](https://developers.bump.sh/).
```
Comment (paulRbr) [inline docs/product-updates/2023-06-07-api-branch-management.md:9]: :tipping_hand_person: suggestion: I would link to the “Branching” endpoints directly

```suggestion
We released our latest feature: Branch Management with the [Bump API](https://developers.bump.sh/group/endpoint-branches).

What do you think?
```

**Review by paulRbr — COMMENTED** [review]
Thanks for this product update that looks great!

I feel there's a bit too much text/details in the explanation of the 4 endpoints. The two first paragraphs are good to “embroid” (pas sûr que ça marche en anglais “broder”) but I don't think we need too much explanation for each endpoint. They are quite simple and self-explanatory.

What do you think?

**Comment by saro0h** [comment]
It make sense to me to not just explain the endpoint, but show to the user the value of it for them. But I understand your point.
I'm openned to suggestions to do rephrase it or maybe remove it (😥).

@jayk-u @antlio Can we get your input please?

File: docs/product-updates/2023-06-07-api-branch-management.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+title: Simplify Your API Workflow with Bump.sh's API New Branch Management Feature
+tags: [New, Bump API]
+image: /files/changelog/temporary-pic.png
+---
+
+[NEED A VISUAL HERE]
+
+We released our latest feature: Branch Management with the [Bump API](https://developers.bump.sh/group/endpoint-branches).
+
+Designed to enhance your API development process, these new endpoints empower API  ecosystem. With the introduction of branch creation, listing, setting default branch, and branch deletion. Bump.sh revolutionizes how you deliver ephemeral versions and begin new branches with major versions. Let's explore how this new feature simplifies your API workflow.
```
[the remark this answers is not in the document]
Comment (saro0h) [reply docs/product-updates/2023-06-07-api-branch-management.md]: Yes you're right. I made changes. Feel free to tell me if it's close to what you want 💁🏽‍♀️

File: docs/product-updates/2023-06-07-api-branch-management.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+---
+title: Simplify Your API Workflow with Bump.sh's API New Branch Management Feature
+tags: [New, Bump API]
+image: /files/changelog/temporary-pic.png
+---
+
+[NEED A VISUAL HERE]
+
+[Branching at Bump.sh](https://docs.bump.sh/help/branching/) makes your API version workflow easier.
```
Comment (scharrier) [inline docs/product-updates/2023-06-07-api-branch-management.md:9]: Should we talk about `version` here?

File: docs/product-updates/2023-06-07-api-branch-management.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+## Latest release
+
+And now, Branch Management is at hand with the [Bump API branching endpoints](https://developers.bump.sh/group/endpoint-branches).
+Designed to enhance your API development process, these new endpoints gives you the opportunity to boost your CI deployments. With the introduction of branch creation, listing, setting default branch, and branch deletion. Bump.sh revolutionizes how you deliver ephemeral versions and begin new branches with major versions.
```
Comment (scharrier) [inline docs/product-updates/2023-06-07-api-branch-management.md:14]: Should we use `revolutionizes`? It feels a bit overpromised. Maybe we could replace by something like `optimize`? WDYT?

File: docs/product-updates/2023-06-07-api-branch-management.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+2. [List of Branches](https://developers.bump.sh/operation/operation-post-docs-parameter-branches):
+Get an overview of all branches associated with your API using the list of branches endpoint. This feature provides a clear and concise view of your branch hierarchy, enabling you to manage and organize them efficiently.
+3. [Set Default Branch](https://developers.bump.sh/operation/operation-patch-docs-parameter-branches-parameter-set_default):
+Setting a default branch for your API documentation is now a breeze. Choose the branch you want to showcase as the default version, and Bump.sh ensures that users accessing your API documentation are directed to the most up-to-date and relevant information.
+4. [Delete Branch](https://developers.bump.sh/operation/operation-delete-docs-parameter-branches-parameter):
+When a branch is no longer needed or has served its purpose, you can easily delete it using the delete branch endpoint. By removing unnecessary branches, you maintain a clean and organized development environment.
```
Comment (scharrier) [inline docs/product-updates/2023-06-07-api-branch-management.md:27]: At first, I would have removed these points, but it does make sense to present the different operations. Maybe we could just have a simple bullet points list with a simple sentence for each point?

**Review by scharrier — COMMENTED** [review]
I left some small comments :)

File: docs/product-updates/2023-06-16-api-branch-management.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
+---
+title: Simplify Your API Workflow
+tags: [New, Bump API]
```
Comment (antlio) [inline docs/product-updates/2023-06-16-api-branch-management.md:3]: What do you think about using this instead? (as we have only one so far)

```suggestion
tags: [New, API]
```

**Comment by saro0h** [comment]
I treated all feedbacks. We just need a picture to illustrate the "branch management" topic @antlio please :)

**Comment by antlio** [comment]
Screenshot 2023-06-23 at 18 19 49

**Review by antlio — APPROVED** [review]
Image and content proposal update added

To be merged unless content feedback

File: static/files/changelog/temporary-pic.png
[the remark this answers is not in the document]
Comment (antlio) [reply static/files/changelog/temporary-pic.png:1]: done! feel free to take over the merge!

**Review by paulRbr — APPROVED** [review]
:rocket: you can merge whenever you want @saro0h

**Pull request merged**